src/crawlee/browsers/_playwright_browser_plugin.py

```diff
@@ -35,26 +35,26 @@ def __init__(
         self,
         *,
         browser_type: BrowserType = 'chromium',
-        browser_options: Mapping[str, Any] | None = None,
-        page_options: Mapping[str, Any] | None = None,
+        browser_launch_options: Mapping[str, Any] | None = None,
+        browser_new_context_options: Mapping[str, Any] | None = None,
         max_open_pages_per_browser: int = 20,
     ) -> None:
         """A default constructor.
 
         Args:
             browser_type: The type of browser to launch ('chromium', 'firefox', or 'webkit').
-            browser_options: Keyword arguments to pass to the browser launch method. These options are provided
+            browser_launch_options: Keyword arguments to pass to the browser launch method. These options are provided
                 directly to Playwright's `browser_type.launch` method. For more details, refer to the Playwright
                 documentation: https://playwright.dev/python/docs/api/class-browsertype#browser-type-launch.
-            page_options: Keyword arguments to pass to the page object is set at the playwright context level.
+            browser_new_context_options: Keyword arguments to pass to the page object is set at the playwright context.
                 These options are provided directly to Playwright's `browser.new_context` method. For more details,
                 refer to the Playwright documentation: https://playwright.dev/python/docs/api/class-browser#browser-new-context.
             max_open_pages_per_browser: The maximum number of pages that can be opened in a single browser instance.
                 Once reached, a new browser instance will be launched to handle the excess.
```

Comment:
Little out of scope, but since we're touching it... Couldn't we set this docstring in the `BaseBrowserPlugin.__init__`? Same for the `BaseBrowserController`.

Reply:
Those abstract classes don't even have `__init__` and that is I think correct. Maybe I misunderstood this point?